### DISCUSSION TITLE
Remove duplicates in cities

### DISCUSSION THREAD

**Discussion by lebroncho** [dr5hn/countries-states-cities-database#221] [General]
Hello, does anyone knows how to filter/remove the duplicates cities or by their name only? Based on this note 
```
You can manually change the spelling mistakes, **duplicates**
or add/edit any records, which are incorrect.
```
I tried this one in vue js ```this.countryCities = [ ...new Set(this.countryCities) ]```

**Comment by MartinsOnuoha** [comment]
@jecfarolan360 If you intend to filter the cities by their names, you can do a needle-haystack search using `Array.filter` on input change. What you have tried above only removes the duplicates in the `countryCities` array. Can you explain what you mean by "filter the duplicates"?

**Comment by lebroncho** [comment]
I'm sorry. I am complicated which is the right term, remove or filter duplicates but my intention is if there is two same city name is showing, I want to show only one in the dropdown/input box.

**Comment by MartinsOnuoha** [comment]
@jecfarolan360 Oh, okay, Since you have the cities stored in `this.countryCities` You can do this on mounted.
```js
mounted() {
  this.countryCities = [ ...new Set(this.countryCities) ]
}
```
You already got it right
